#!/usr/bin/python3


def raise_exception():
    try:
        pass
    raise TypeError:

